graylog2-web-interface/src/views/components/sidebar/FieldList.jsx

```diff
@@ -1,184 +0,0 @@
-import React from 'react';
```

Comment:
It might make sense to move a file instead of deleting it (especially in this case, where we are still keeping lots from the component, just under a new name), so we retain its history.

Reply:
That is a good point I did not thought about. I named the component `FieldsOverview` because I wanted to avoid confusion with the `List` component in the same directory.

Reply:
It looks like I have to create a new PR to tell git I moved the file and changed the name. At least this seems to be the easiest way

Reply:
I created a new branch and used `git mv` to move the and rename the file. This seems to work when looking at the commit: https://github.com/Graylog2/graylog2-server/commit/a74d2e435f218ffef7fac8fab212bbfe2ff7b6b0
But not when looking at the PR..
https://github.com/Graylog2/graylog2-server/compare/refactor-field-list-v2?expand=1